cluster-provision/k8s/1.19/node01.sh

```diff
@@ -2,9 +2,6 @@
 
 set -ex
 
-# Ensure that hugepages are there
-cat /proc/meminfo | sed -e "s/ //g" | grep "HugePages_Total:64"
```

Comment:
Pedantic: `tr -d ' '`? :)

Reply:
Yep, would have done the trick. Unfortunately these 2 lines are no longer there... they were casualties to this PR :smile: (or should be if I missed them in any of the other scripts :smile:  )

Reply:
Facepalm, ignore me, time for the week-end! :)